### PR TITLE
improvements to STDIO, looking to remove the syslog emittor

### PR DESCRIPTION
**Pull request by chgray**
* Improving STDIO emission, looking to remove syslog emission (this will be more obvious/interesting in the next PR)
* make failure clearer when whitespace chars are given in variable encoding syntax
* add printf to windows and linux CI/CD testing - in prep of removing syslog emission

### CODE REVIEW

File: src/clog/TraceEmitterModules/CLogSTDOUT.cs [hunk 1]
```diff
@@ -134,7 +134,7 @@ public void TraceLineDiscovered(string sourceFile, CLogOutputInfo outputInfo, CL
                         printf += "%S";
                         break;
                     case CLogEncodingType.Pointer:
-                        printf += "%p";
+                        printf += "0x%llx";
```
Comment (nibanks) [inline src/clog/TraceEmitterModules/CLogSTDOUT.cs:137]: But... doesn't this depend on pointer size?

File: .github/workflows/check-clog-linux-printf.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Build CLOG and run Tests
+      run: ./runTests_stdio.ps1
+      working-directory: ./examples
+      shell: pwsh
+
+
```
Comment (nibanks) [inline .github/workflows/check-clog-linux-printf.yml:26]:
```suggestion
      shell: pwsh
```

**Review by nibanks — DISMISSED** [review]
Ship it

File: src/clog/TraceEmitterModules/CLogSTDOUT.cs (hunk 1 above)
Comment (chgray) [reply src/clog/TraceEmitterModules/CLogSTDOUT.cs:137]: the worry is x86?  this should be okay no?

**Review by nibanks — APPROVED** [review]

**Pull request merged**